### PR TITLE
Fix notifications test description

### PR DESCRIPTION
**Pull request by wlsf82**
I was exploring the project and found these missing words, so I decided to fix it and propose the change.

### CODE REVIEW

**Comment by wlsf82** [comment]
Hey, @bahmutov, would you mind looking into this small contribution, please?

File: cypress/tests/ui/notifications.spec.ts [hunk 1]
```diff
@@ -81,7 +81,7 @@ describe("Notifications", function () {
       cy.visualSnapshot("Notification count after notification dismissed");
     });
 
-    it("User C likes a transaction between User A and User B; User B and get notifications that User C liked transaction", function () {
+    it("User C likes a transaction between User A and User B; User B and A get notifications that User C liked transaction", function () {
```
Comment (wlsf82) [inline cypress/tests/ui/notifications.spec.ts:84]:
```suggestion
    it("User C likes a transaction between User A and User B; User A and User B get notifications that User C liked transaction", function () {
```

**Comment by wlsf82** [comment]
Could someone review this, please?

**Review by kevinold — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**